gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GradebookUiSettings.java

```diff
@@ -78,4 +80,12 @@ public void setCategoryScoreVisibility(final String category, final Boolean visi
 	public String toString() {
 		return ToStringBuilder.reflectionToString(this, ToStringStyle.MULTI_LINE_STYLE);
 	}
+
+	public void setCategoryColor(String category, String color) {
+		categoryColors.put(category, color);
+	}
+
+	public String getCategoryColor(String category) {
```

Comment:
Is this name or id? Sorry to be a pain, but could it be made more explicit as to which one, or just add doco?